### PR TITLE
testsuite for v.random.strat.sampling

### PR DESCRIPTION
**Pull request by neteler**
Test case for `v.random.strat.sampling`.

Note that this PR depends on #6 being merged first.

TODO: check why 78 is right in the result. Probably after merge of #6 the value 38 is expected and right.

### CODE REVIEW

File: testsuite/test_v_random_strat_sampling.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @classmethod
+    def tearDownClass(cls):
+        """Remove the temporary region and generated data"""
+        cls.del_temp_region()
+
+    def test_points(self):
+        """Test divide points"""
+        # note: numeric ZIPNUM column works
+        self.assertModule('v.random.strat.sampling', input=self.invect, column='NAME',
+                          output=self.outtrain, npoints=2)
```
Comment (anikaweinmann) [inline testsuite/test_v_random_strat_sampling.py:38]: The created vector map must also be deleted again, right?

File: testsuite/test_v_random_strat_sampling.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        self.assertVectorExists(self.outtrain)
+        # there are 19 names in the table: 19 * unclear = 78
```
Comment (anikaweinmann) [inline testsuite/test_v_random_strat_sampling.py]: I get 38 points, but I merged #6 first

Comment (anikaweinmann) [reply testsuite/test_v_random_strat_sampling.py]:
```
FAIL: test_points (__main__.Testdivide)
Test divide points
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test.py", line 40, in test_points
    self.assertVectorFitsTopoInfo(self.outtrain, topology)
  File "/home/abettge/repos/grass78/dist.x86_64-pc-linux-gnu/etc/python/grass/gunittest/case.py", line 361, in assertVectorFitsTopoInfo
    precision=0)
  File "/home/abettge/repos/grass78/dist.x86_64-pc-linux-gnu/etc/python/grass/gunittest/case.py", line 254, in assertModuleKeyValue
    self.fail(self._formatMessage(msg, stdMsg))
AssertionError: v.info map=training layer=1 -t difference:
mismatch values (key, reference, actual): [('points', 78, 38)]
command: v.info map=training layer=1 -t {'map': 'training', 'flags': 't'}
```

**Review by anikaweinmann — CHANGES_REQUESTED** [review]

**Review by anikaweinmann — APPROVED** [review]

File: testsuite/test_v_random_strat_sampling.py (hunk 1 above)
Comment (neteler) [reply testsuite/test_v_random_strat_sampling.py:38]: Yes, good catch

**Pull request merged**